### PR TITLE
translate: rebrand

### LINKED ISSUE #1884: Branding updates required for Translate API client libraries
On November 15, 2016, Google launch a rebranding of Google Translate API to Google Cloud Translation API. This does not impact the method names but the URLbase as well as readmes, samples and comments.

URLbase has been already updated to translation.googleapis.com

We need to have update the Client library, the readme, client library docs, samples and code comments to use "Google Cloud Translation API" instead of Google Translate API. The short version of the new name is "Cloud Translation API" or "Translation API"

Note: the product Google Translate still exists. It is the API Google Translate API that has been re-branded.

### PR DESCRIPTION
**Pull request by stephenplusplus**
Fixes #1884

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -796,6 +796,75 @@ fs.createReadStream('./audio.raw')
...
+```
+$ npm install --save google-cloud
+```
+
+```js
+var gcloud = require('google-cloud');
+var translate = gcloud.translate;
+```
+
+#### Using the Google Translate API module
```
Comment (callmehiphop) [inline README.md]: Should this be `Translation`? And should we be publishing a new packages called `@google-cloud/translation`?

File: README.md [hunk 2]
```diff
@@ -796,6 +796,75 @@ fs.createReadStream('./audio.raw')
...
+
+#### Using the all-in-one module
+
+```
+$ npm install --save google-cloud
+```
+
+```js
+var gcloud = require('google-cloud');
+var translate = gcloud.translate;
```
Comment (callmehiphop) [inline README.md]: Should we rename the method `translation`?

**Comment by stephenplusplus** [comment]
RE: module and method name changes, I don't believe so, because of this part of #1884:

>  This does not impact the method names but the URLbase as well as readmes, samples and comments.

**Comment by callmehiphop** [comment]
@stephenplusplus I assumed when they mentioned method names that was for the `translate` module. For the umbrella package we have a method specifically named `translate` which is what I'm wondering about

**Comment by stephenplusplus** [comment]
That's funny, I thought it may have meant the reverse, since his note specifically mentioned "method names" and `gcloud.translate` is one :) But enough guessing, @danoscarmike, can you help us find some clarity on this:

- do we change the module name? npm install google-cloud/translate OR npm install google-cloud/translation

- do we change the method name? require('google-cloud').translate(...) OR require('google-cloud').translation(...)

**Comment by stephenplusplus** [comment]
@callmehiphop sounds like we're keeping the names. Let's merge this 🐶 ?

**Pull request merged**